pkg/apis/pipeline/constant.go

```diff
@@ -19,4 +19,13 @@ package pipeline
 const (
 	// TektonReservedAnnotationExpr is the expression we use to filter out reserved key in annotation
 	TektonReservedAnnotationExpr = "(results.tekton.dev|chains.tekton.dev)/.*"
```

Comment:
Does it still make sense to ignore all other results annotations?

In addition, if results changes their implementation of which annotations are used, will we need to update pipelines as well? Does this create coupling between the two projects that could lead to version incompatibilities?

Reply:
I think we can just remove `results.tekton.dev` annotations from now. The initial issues was only for `chains.tekton.dev` so.. let's only manages this.

@khrm do you have a list of annotations `results` are using **and** when (creation, …) ?

Reply:
The annotations I put are the only one which are used during creation time.

Reply:
I think we should go with writing a CEL admission controller to resolve this instead of introducing a coupling between chains and results.

/cc @tektoncd/chains-collaborators I think chains should also do the same.
@tektoncd/results-collaborators